### PR TITLE
[cli] New CLI command 'log' to change log level.

### PR DESCRIPTION
**Pull request by EskoDijk**
Here is a separate PR for the 'log' CLI command that I use.  It seems a good idea to separate features not related to the radiomodel feature as separate PRs as suggested by @simonlingoogle . 

It can be tested by e.g. entering the command "log debug" and then do an action like sending a ping or adding a node.

### CODE REVIEW

File: cli/CmdRunner.go [hunk 1]
```diff
@@ -623,6 +625,14 @@ func (rt *CmdRunner) executeWeb(cc *CommandContext, webcmd *WebCmd) {
 	}
 }
 
+func (rt *CmdRunner) executeLogLevel(cc *CommandContext, cmd *LogLevelCmd) {
+	if len(cmd.Level) == 0 {
```
Comment (simonlingoogle) [inline cli/CmdRunner.go:629]:
```suggestion
	if cmd.Level == "" {
```

**Review by simonlingoogle — APPROVED** [review]
LGTM 👍 

with a nit

**Comment by simonlingoogle** [comment]
@jwhui Would need https://github.com/openthread/ot-ns/pull/324 to resolve CI fails.

**Comment by simonlingoogle** [comment]
Can we add a method to get/set log level in OTNS.py @EskoDijk

**Comment by EskoDijk** [comment]
> Can we add a method to get/set log level in OTNS.py @EskoDijk

I added a str property in OTNS.py.

**Comment by simonlingoogle** [comment]
> > Can we add a method to get/set log level in OTNS.py @EskoDijk
> 
> I added a str property in OTNS.py.

👍 Great idea to use `@property`.

**Pull request closed without merging**